scripts/makepbcore.py

```diff
@@ -886,6 +893,9 @@ def main(args_):
         ififuncs.sha512_update(sha512_manifest, csv_filename)
         print((' - Updating %s with %s' % (sha512_manifest, csv_filename)))
         print(metadata_error)
+    if failed_path:
+        print("***** [missing_metadata*] Some metadata cannot be recognised by the script.\n\tThe source path(s) shows below.\n\tRepeated locations means multiple missing matadata.")
```

Comment:
```suggestion
        print("***** [missing_metadata*] Metadata cannot be recognised by the script.\n\tThe source path(s) are shown below.\n\tRepeated locations mean the absence of metadata in multiple fields.")
```

Reply:
Thanks! This has been fixed in the latest 2 commits.

Reply:
It looks like you could have committed this suggestion directly and then niamh would have been credited with the commit? Looks like a newer GitHub feature.

Reply:
Thanks! So it will commit what review changes? Sounds helpful. We had a further discussion for the re-phrasing after this so will do that next time.

Reply:
It appears to be the case - I just saw the option there but I have not used it.

Reply:
Btw where possible it would be best to allow reviewers to make changes this way, then they get commits and contributer credits. If you look at the commit history, most ifi staff members who worked in digital all have commits over the years, it’s a great way to encourage contributions